packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -357,37 +357,46 @@ function WorklistDataGrid(props) {
     }
   };
 
-  const selectMultipleRows = useCallback((newEndRowIndex, rangeInfo) => {
-    if (!onRowSelect || !rangeInfo) {
-      return;
+  const getRowsInRangeToUnselect = useCallback((startOfRange, prevEndOfRange, newEndOfRange) => {
```

Comment:
This method is only used in one place. It seems like the logic is small enough to just be included in the caller and this method removed.

Reply:
We have kept the method separate for readability.